src/game/client/swarm/gameui/swarm/rd_workshop_frame.cpp

```diff
@@ -370,7 +370,7 @@ void ReactiveDropWorkshop::OnCommand( const char *command )
 		m_pTxtEditingTitle->GetText( szTitle, sizeof( szTitle ) );
 		if ( !*szTitle )
 		{
-			V_snprintf( szTitle, sizeof( szTitle ), "Untitled Addon" );
+			V_snprintf( szTitle, sizeof( szTitle ), "#rd_workshop_untitled_addon_title" );
```

Comment:
This is being sent directly to the Steam Community website, which doesn't use the translation files from Alien Swarm: Reactive Drop.